app/javascript/packs/stylesheets/analytics.scss

```diff
@@ -72,6 +72,14 @@
   height: 50px; // the height of each row
 }
 
+.analytics-table-flow {
```

Comment:
Not sure this is needed, it just adds whitespace to the top of the table which isn't really necessary, especially because it makes it uneven with the table next to it

Reply:
Also for the sake of alignment. Ongoing discussion to resolve with PO team in Slack.

Reply:
Removed https://github.com/SaraAlert/SaraAlert/pull/834/commits/29694316e2a04f2c79c63780fbeafa6ea9e19dbb